_pages/index.html

```diff
@@ -0,0 +1,114 @@
+        of and instead fling open the doors, welcoming you to the big, wide
+        world of computing. Oh, and did we mention the challenge of concurrency
+        and solving asynchronous problems, so that your program can take
+        advantage of the multi-core CPU inside each machine?
+
+      </section>
+      </section>
+        <section class="card-body course-information">
+            <section class="flexbox">
+                <section class="flexbox-item">
+                    <h2>
+                        Time and Location
+                    </h2>
+                    <p>
+                        <strong>Instructor: </strong>Lawrence Angrave (angrave)
+                    </p>
+                    <p>
+                        <strong>Lecture: </strong>MWF 8am-8.50am
+                    </p>
+                    <p>
+                    </p>
+                    <p>
+                        <strong>Location: </strong>ECEB 1002
+                    </p>
+                </section>
+                <section class="flexbox-item">
+                    <h2>
+                        Peer Tutoring
+                    </h2>
+                    <p>
+                    <p>
+                        You can find a full list of times and locations on the <a href="/peer_tutoring.html">calendar</a>.
+                    </p>
+                    <p>
+                        <a
+                        href="https://piazza.com/illinois/spring2019/cs241"
+                        class="btn btn-primary"
+                        role="button"
+                        >Piazza</a>
+                        </p>
+                        <p>
+                            Please read our <a href="http://localhost:4000/resources.html#piazza-search-first-dont-post">Piazza policy, before posting.</a>
+                        </p>
+                </section>
+                <section class="flexbox-item">
+                    <h2>
+                        Grading Policy
+                    </h2>
+                    <p>
+                        <strong>Machine Problems: </strong>35% (weekly)
+                    </p>
+                    <p>
+                        <strong>Labs: </strong>12% (weekly, lowest lab dropped)
+                    </p>
+                    <p>
+                        <strong>Lab Attendance:</strong> 3% (2 drops)
+                    </p>
+                    <p>
+                        <strong>Quizzes: </strong> 25% (8 quizzes, in the CBTF)
+                    </p>
+                    <p>
+                        <strong>Final Exam: </strong>25% (comprehensive)
+                    </p>
+                </section>
+                <section class="flexbox-item">
+                    <h2>
+                        Course Prerequisites
+                    </h2>
+                    <p>
+                        <strong>CS 225: </strong>Data Structures
```

Comment:
would be cool if we could have links to these courses. We should be able to use the course.engr.whatever versions of the links to get a semester agnostic link to their current homepage.

Reply:
alternatively, have a link to a snapshot of these course pages from some semester

Reply:
There is a semester agnostic link. Why do you think this is useful?

Reply:
It doesn't seem like these are links at all. It's just plain text

Reply:
courses.engr.illinois.edu/cs225
courses.engr.illinois.edu/cs233